unired-tag/config.js

```diff
@@ -1,12 +1,17 @@
 const {
-  constants: { TIMEFRAMES },
+  constants: { AUTH_TYPE_KEYS, TIMEFRAMES },
 } = require('@notify-watcher/core');
 
 const config = {
   name: 'unired-tag-total',
   displayName: 'Unired Tag Total',
   description: 'Watch your tag ballot using Unired',
-  auth: true,
+  auth: {
+    rut: {
+      type: AUTH_TYPE_KEYS.string,
+      description: 'Your RUT in the format 12345678-K (no points)',
```

Comment:
Eventually it could have a `cleanRUT` function that ensures that even though a RUT with points is submitted, it is cleaned up and treated as one with no points for example

Reply:
yeah it could! actually, this is a js object, so we could add a function that process the field directly into the config